src/test/java/MorseCodeConverterTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MorseCodeConverterTest {
+
+    @Test
+    public void testFillMorse() {
```

Comment:
You don't need to start your test method names with the word test. This is an old convention that isn't used anymore.